ui/selectors/selectors.js

```diff
@@ -742,6 +743,30 @@ export function getShowExtensionInFullSizeView(state) {
   return Boolean(showExtensionInFullSizeView);
 }
 
+export function getSmartTransactionsOptInStatus(state) {
+  return sharedSelectors.getSmartTransactionsOptInStatus(state);
+}
+
+export function getIsAllowedStxChainId(state) {
+  return sharedSelectors.getIsAllowedStxChainId(state);
+}
+
+export function getSmartTransactionsEnabled(state) {
+  return sharedSelectors.getSmartTransactionsEnabled(state);
+}
+
+export function getIsSmartTransaction(state) {
+  return sharedSelectors.getIsSmartTransaction(state);
+}
+
+export function getIsStxOptInAvailable(state) {
+  return sharedSelectors.getIsStxOptInAvailable(state);
+}
+
+export function getFeatureFlagsByChainId(state) {
```

Comment:
why are we doing this? why not just the selectors from the sharedSelectors directly in the code?